k9mail-library/src/test/java/com/fsck/k9/mail/K9LibRobolectricTestRunner.java

```diff
@@ -0,0 +1,20 @@
+package com.fsck.k9.mail;
+
+import org.junit.runners.model.InitializationError;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+public class K9LibRobolectricTestRunner extends RobolectricTestRunner {
+
+    public K9LibRobolectricTestRunner(Class<?> testClass) throws InitializationError {
+        super(testClass);
+    }
+
+    @Override
+    protected Config buildGlobalConfig() {
+        return new Config.Builder()
+                .setSdk(21)
```

Comment:
I would use the ``targetSdkVersion`` which is 22. This is also the default of robolectric if we do not configure anything.

So why even configure a sdk version here? In the past robolectric did not provide all the sdk versions, they were lagging behind Android releases. I would keep the explicit configuration to make it less depended on changes in the build.gradle.

I added a commit to set sdk to 22 in both runners.

You can explicitly override the runner config by adding an annotation again, but before each test had the annotation to use sdk 21, which violates DRY.